### PR TITLE
docs(repo): linting with ssgs

### PR DESCRIPTION
**Pull request by pamgoodrich**
Fixes https://github.com/stoplightio/platform-docs/pull/752.

Added note about hosted style guide updates and Spectral CLI.

**Checklist**

- [ ] Tests added / updated
- [X] Docs added / updated

**Does this PR introduce a breaking change?**

- [ ] Yes
- [X] No

### CODE REVIEW

File: docs/guides/2-cli.md [hunk 1]
```diff
@@ -60,6 +60,8 @@ Here you can build a [custom ruleset](../getting-started/3-rulesets.md), or exte
 - [OpenAPI ruleset](../reference/openapi-rules.md)
 - [AsyncAPI ruleset](../reference/asyncapi-rules.md)
 
+> If you use rules created or updated in a hosted [Stoplight API project](https://docs.stoplight.io/docs/platform/branches/pam-716-updated-landing-page/c433d678d027a-create-rules) with the Spectral CLI, you must publish the project from Stoplight before the updates are using for linting.
```
Comment (dotslashderek) [inline docs/guides/2-cli.md]: "are using for linting" -> " are used for linting" :)

**Review by dotslashderek — DISMISSED** [review]

**Review by P0lip — CHANGES_REQUESTED** [review]
I don't think we should be including a style guide in Spectral project itself.
This is likely to cause confusion for people.
Thoughts @mnaumanali94?

The docs change is OK though.

**Review by P0lip — DISMISSED** [review]

**Comment by pamgoodrich** [comment]
> I don't think we should be including a style guide in Spectral project itself. This is likely to cause confusion for people. Thoughts @mnaumanali94?
> 
> The docs change is OK though.



> 



> I don't think we should be including a style guide in Spectral project itself. This is likely to cause confusion for people. Thoughts @mnaumanali94?
> 
> The docs change is OK though.

Good point, Jakub. Should I remove both the styleguide.json and .spectral.mjs files?

**Comment by P0lip** [comment]
> Good point, Jakub. Should I remove both the styleguide.json and .spectral.mjs files?

I took care of that already :)

**Pull request merged**